tests/packagedcode/data/podspec/package/SwiftLib.podspec

```diff
@@ -0,0 +1,140 @@
+#
```

Comment:
Is this SwiftLib.podspec the example from their doc? It feels a little contrived. Can you find a better set of data files?